source/code/plugin/KubernetesApiClient.rb

```diff
@@ -210,6 +211,123 @@ def getContainerLogsSinceTime(namespace, pod, container, since, showTimeStamp)
                 end
                 return containerLogs
             end
+
+            def getContainerResourceRequestsAndLimits(metricJSON, metricCategory, hostName, metricNameToCollect, metricNametoReturn)
```

Comment:
Minor edit but would you consider renaming getContainerResourceRequestsAndLimits to parse* since the actual get is happening in in_kube_perf.rb